assets/src/design-system/icons/index.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+
+export {
+  Add,
+  AddFilled,
+  AddOutlined,
+  Bookmark,
+  Checkmark,
+  Edit,
+  Grid,
+  Info,
+  KeyboardShortcut,
+  List,
+  Lock,
+  Minus,
+  Plus,
+  Preview,
+  Search,
+  Settings,
+  Trash,
+  Unlock,
+} from './action';
```

Comment:
Any particular reason these are not just `export * from ...`?

Reply:
initially it was because i was tracking imports to make sure I had all the icons, but that's a great idea to clean it up here in the index. I'll update this throughout!